content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -536,7 +579,7 @@ If multiple resources are matched in `spec.matchConstraints`, all of matched res
 For example, the following policy definition 
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?